package.json

```diff
@@ -1,20 +1,19 @@
 {
   "name": "contexture-export",
-  "version": "1.0.0",
+  "version": "1.0.1",
   "description": "Contexture Exports",
   "main": "lib/contexture-export.js",
   "scripts": {
     "build": "webpack",
     "danger": "duti",
-    "lint": "eslint src/*.js test/*.js",
+    "lint": "eslint --ignore-path .gitignore 'src/**/*.js'",
```

Comment:
I think this skips tests, but we should lint them as well. We should just run this across all js, regardless of path.

Reply:
There is no tests directory anymore 🤷